### ISSUE TITLE
Quoted headers break CSV import

### ISSUE THREAD

**Issue by TomyLobo** [shridhar-tl/jira-assistant#317]
### Checklist before you being

- [X] I am sure that I am already using latest version of Jira Assistant
- [X] I had verified that, no similar issues are already reported in [issue tracker](https://github.com/shridhar-tl/jira-assistant/issues)
- [X] I had tried reproducing this issue with with [Web version of Jira Assistant](https://app.jiraassistant.com) to ensure that this issue is not recently fixed and yet to be published
- [X] I had verified that, my query is not answered in [FAQ section of website](https://www.jiraassistant.com/faq)
- [X] I had verified that, my issue is not listed as [known bugs in website](https://www.jiraassistant.com/version-history)

### How do you use Jira Assistant?

Browser extension

### Are you using cloud version of Jira or self hosted (data center / server) of Jira.

Self hosted (datacenter)

### Version of Jira Assistant

v2.54

### What browser are you using?

Firefox

### What OS are you using? You need not disclose this if you feel it is irr-relevant for your issue.

Linux

### Bug Description

If you quote the headers, CSV import won't pick up the fields.

Example CSV:
```csv
"Ticket No","Start Date","Timespent","Comment"
"JA-1001","15-Aug-2023 18:12:11","1w 2d 3h 4m","Logs 59 hours and 4 mins"
"JA-1001","15-Aug-2023 18:12:11","1d 1h","Logs 9 hours"
"JA-1002","15-Aug-2023 18:12:11","12.5","Logs 12 hours and 30 mins"
"JA-1003","15-Aug-2023 18:12:11","14:4","Logs 14 hours and 40 mins"
"JA-1003","15-Aug-2023 18:12:11","8","Logs 8 hours"
```
Expected result:
![image](https://github.com/shridhar-tl/jira-assistant/assets/963797/8f052f95-51cc-49da-9e7f-c23eb91b96ba)

Actual result:
![image](https://github.com/shridhar-tl/jira-assistant/assets/963797/231b929c-21a1-403c-acc1-34761b143b67)

### Checklist before you submit

- [X] I have ensured not to paste any confidential information like Jira url, Mail id, etc.
- [X] I have verified my browser console logs and added necessary details (if required)
- [X] I have added required screenshots if the bug is related to UI (as necessary)

**Comment by TomyLobo** [comment]
I think this worked fine until recently, but I'm not 100% sure.

**Comment by shridhar-tl** [comment]
Hi @TomyLobo , this issue has been resolved in JA v2.55 and would be available for extension uses after next publish.

If you want to look at the fix immediately, you can have a look at it via web version using following url:

Web Version of JA: https://app.jiraassistant.com/?authType=1

You need not do any separate authentication in web version. Still authentication is taken care by Extension itself while web is just used to display the page. So Web helps you use latest code with bug fixes without compromising on any aspects.

You can also permanently switch to Web so that just in case of such issues, you will know in advance even before I publish the extension so that the issue can be reported and immediately I would be able to resolve it.

Once the changes are published to extension, it takes time to push another version due to various limitations.

To know more about Web version, you can have a look into following Urls:
https://www.jiraassistant.com/install-options/
https://www.jiraassistant.com/why-web-version/

As of now closing this ticket considering this issue is resolved. Please feel free to reopen it if you feel the issue is still not resolved.
